lib/i18n/tasks/ar/config.rb

```diff
@@ -13,7 +13,7 @@ def self.reset
 
           if file
             config = YAML.load_file(file)
-            self.locales = config['locales']
+            self.locales = config['locales'] rescue ['en']
```

Comment:
Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.